store/kv/measure.go

```diff
@@ -216,5 +277,6 @@ func (m *TxImplWithMetrics) ReadRange(ctx context.Context, table []byte, lkey Ke
 		it, err = m.tx.ReadRange(ctx, table, lkey, rkey, isSnapshot)
 		return err
 	})
+	// Read bytes are counted in the iterator
```

Comment:
Yes, I think so. When people look through the operations, it's helpful to know that this is counted inside the iterator, because this is always a streaming request, even if you just do a read one at the high level.